base/linalg/dense.jl

```diff
@@ -614,7 +614,7 @@ triangular factor.
 
 # Examples
 ```jldoctest
-julia> A = 2.7182818 * eye(2)
+julia> A = Matrix(2.7182818I, 2)
```

Comment:
The `I` looked like a `1` at first sight and I was confused... Perhaps add an explicit `*`? `Matrix(2.7182818*I, 2)`

Reply:
Revised accordingly on push. Thanks!